addons/hw_drivers/tools/helpers.py

```diff
@@ -472,6 +472,14 @@ def read_file_first_line(filename):
         with path.open('r') as f:
             return f.readline().strip('\n')
 
+
+def restart_odoo_or_reboot(action, delay=3):
+    _logger.info("restart_odoo_or_reboot called with %s as action", action)
+    if action == "restart_odoo":
+        odoo_restart(delay)
+    else:
+        subprocess.call(["sudo", "reboot"])
```

Comment:
was there not something related to the usage of subprocess.run instead of subprocess.call ? Might be also a good idea to add a check if it is not a windows computer (even if functionally this code can't be reached)